From: Mike Dacre <mike.dacre@gmail.com>
Date: Fri, 11 Dec 2015 15:42:26 -0800
Subject: Altered find() functions for python3 compatibility

Changed the char *name='' entries in the four find() definitions
containing them to be simple name=''.
This fixed the following error:
    TypeError: expected bytes, str found

(cherry picked from commit 8f2465435fdb54bbe12586a1158cb3c51ad45622)
---
 pyslurm/pyslurm.pyx | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/pyslurm/pyslurm.pyx b/pyslurm/pyslurm.pyx
index 75e6f07..2a7cf17 100644
--- a/pyslurm/pyslurm.pyx
+++ b/pyslurm/pyslurm.pyx
@@ -684,7 +684,7 @@ cdef class partition:
 
 		return self._PartDict.get(partID, {})
 
-	def find(self, char *name='', val=''):
+	def find(self, name='', val=''):
 
 		u"""Search for a property and associated value in the retrieved partition data.
 
@@ -1758,7 +1758,7 @@ cdef class job:
 
 		return self._JobDict.get(jobID, {})
 
-	def find(self, char *name='', val=''):
+	def find(self, name='', val=''):
 
 		u"""Search for a property and associated value in the retrieved job data.
 
@@ -3170,7 +3170,7 @@ cdef class reservation:
 
 		return self._ResDict.get(resID, {})
 
-	def find(self, char *name='', val=''):
+	def find(self, name='', val=''):
 
 		u"""Search for a property and associated value in the retrieved reservation data
 
@@ -3556,7 +3556,7 @@ cdef class block:
 
 		return self._BlockDict.get(blockID, {})
 
-	def find(self, char *name='', val=''):
+	def find(self, name='', val=''):
 
 		u"""Search for a property and associated value in the retrieved block data.
 
